test/jdk/java/lang/management/ThreadMXBean/Locks.java

```diff
@@ -61,10 +64,12 @@ private static void assertNoLock(Thread t) {
             return;
         }
         String name = t.getName();
+        // Carrier Thread can hold a lock on a VirtualThread, which we ignore.
```

Comment:
This comment's placement can make the user think the whole point of the code that follows is to ignore virtual threads here. Maybe it would be better if placed right before the added filter() call.